src/tests/gfx-cube.edn

```diff
@@ -49,7 +48,7 @@
     .time (Math.Multiply 0.7) (Math.AxisAngleY) (Math.Rotation) >= .rotY
     .time (Math.Multiply 0.9) (Math.AxisAngleZ) (Math.Rotation) >= .rotZ
     .rotX (Math.MatMul .rotY) (Math.MatMul .rotZ) > .transform
-    .drawable (GFX.Draw)
+    (GFX.Drawable :Transform .transform :Mesh .mesh) (GFX.Draw)
```

Comment:
hmm to be honest I'd like to have transform as input here
simply cos you can do something very nice like
```clojure
.transform
(GFX.Drawable .home)
(GFX.Drawable .fence)
(GFX.Drawable .chair)
...
```
Very Shards, and very real if pivot is proper